test/simulator/moves/electricterrain.js

```diff
@@ -70,6 +70,7 @@ describe('Electric Terrain', function () {
 		battle = BattleEngine.Battle.construct();
 		battle.join('p1', 'Guest 1', 1, [{species: "Jolteon", ability: 'voltabsorb', moves: ['electricterrain', 'rest']}]);
 		battle.join('p2', 'Guest 2', 1, [{species: "Pidgeot", ability: 'keeneye', moves: ['doubleedge', 'rest']}]);
+		battle.seed = [0, 0, 0, 0];
```

Comment:
I assume a crit Double Edge could break this test otherwise?
We should use a weaker move instead in that case.